admin/c2cgeoportal_admin/views/layers_wms.py

```diff
@@ -84,7 +98,15 @@ def view(self):
                  request_method='POST',
                  renderer='../templates/edit.jinja2')
     def save(self):
-        return super().save()
+        response = super().save()
+        parent_id = self._appstruct['parent_id'] \
+            if hasattr(self, '_appstruct') and self._appstruct is not None \
+            and self._appstruct.get('parent_id') else None
+        if parent_id:
+            parent = self._request.dbsession.query(TreeGroup).get(parent_id)
+            rel = LayergroupTreeitem(parent, self._obj, 100)
+            self._request.dbsession.add(rel)
+        return response
 
```

Comment:
as layerWmsView, WmtsView, treeGroupView are TreeItemViews, this customized save could be declared at TreeItemView level.

Reply:
Ok